chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/TestFunctionality.java

```diff
@@ -52,4 +60,32 @@ public String getYear() {
     public void setYear(final String year) {
         this.year = year;
     }
+
+    /**
+     * @return the certificationCriterion
```

Comment:
I'm not sure if we require JavaDoc for getters and setters. Happy to have them, but don't know if they're necessary. If we _do_ have them, they should probably have the right format; I'm specifically thinking of the warning I always get about "the first sentence must end with a period"